From 0b5a32b21021c356d782782b6b0b2e84d4b2f61e Mon Oct 13  00:57:15 2025
From: jhuang6451 <xplayerhtz123@outlook.com>
Date: Mon, 13 Oct 2025 00:57:15 +0000
Subject: [PATCH] dts: qcom: sm8150: nabu: Add panel rotation property

The physical orientation of the panel is portrait, while the
intended default usage is landscape. Add the rotation property
to correct the display orientation in early boot stages like
the framebuffer console.

---
 arch/arm64/boot/dts/qcom/sm8150-xiaomi-nabu.dts | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/boot/dts/qcom/sm8150-xiaomi-nabu.dts b/arch/arm64/boot/dts/qcom/sm8150-xiaomi-nabu.dts
index d79d2b5fd..9072c69f7 100644
--- a/arch/arm64/boot/dts/qcom/sm8150-xiaomi-nabu.dts
+++ b/arch/arm64/boot/dts/qcom/sm8150-xiaomi-nabu.dts
@@ -614,6 +614,7 @@ display_panel: panel@0 {
 		vddio-supply = <&vreg_l14a_1p8>;
 		reset-gpios = <&tlmm 6 GPIO_ACTIVE_LOW>;
 		backlight = <&backlight>;
+		rotation = <90>;
 
 		ports {
 			#address-cells = <1>;